gordon-plugin/src/main/kotlin/com/banno/gordon/TestDistributor.kt

```diff
@@ -158,6 +163,12 @@ private fun CoroutineScope.runTestsInPool(
     }
 }
 
+private fun getProgress(currentTest: Float, totalTests: Int?): String {
+    return totalTests?.let{
+        "%s %s Complete".format("${currentTest.toInt()}/$totalTests", "${(currentTest / totalTests) * 100}%")
+    } ?: "Retrying..."
+}
```

Comment:
I think I would simplify this significantly:

```suggestion
private fun getProgress(currentTest: Int, totalTests: Int?) =
    totalTests?.let { "$currentTest/$totalTests" } ?: "Retrying"
```